creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -695,6 +726,13 @@ async function _secondarySyncFromPrimary({
   tracing.setSpanAttribute('result', result)
   tracing.setSpanAttribute('mode', mode)
 
+  secondarySyncFromPrimaryLogger = createChildLogger(
+    secondarySyncFromPrimaryLogger,
+    {
+      syncAllStepsDuration: Date.now() - start
+    }
+  )
+
```

Comment:
these changes were all part of an earlier PR so not sure why they're showing up here, but it's just adding durations to logs and was tested

Reply:
weird